executor/new_executor_xapi.go

```diff
@@ -257,25 +260,39 @@ func (e *NewXSelectIndexExec) nextForDoubleRead() (*Row, error) {
 func (e *NewXSelectIndexExec) fetchHandles(idxResult xapi.SelectResult, ch chan<- *lookupTableTask) {
 	defer close(ch)
 
-	workCh := make(chan *lookupTableTask, 10)
+	workCh := make(chan *lookupTableTask, 1)
```

Comment:
We don't need many buffer. 

Every time sending to channel may block, a new goroutine is created.
At line 290:

```
		select {
			case workCh <- task:
			default:
				go e.pickAndExecTask(workCh)
				concurrency++
				workCh <- task
		}
```

I'm considering not reuse worker and `go goroutine()` directly for each `lookupTableTask`, but I'm not sure.